src/Sensors/RequestSensor.php

```diff
@@ -78,6 +91,8 @@ public function __invoke(Request $request, Response $response): array
                 statusCode: $response->getStatusCode(),
                 requestSize: strlen($request->getContent()),
                 responseSize: $this->parseResponseSize($response),
+                payload: rescue(fn () => $request->getPayload(), report: false),
```

Comment:
I went with `getPayload` because it exclude other input such as query string parameters, which are already captured.

The only downside is that if someone sends invalid JSON, we won't capture anything. An alternative would be to capture the raw string in the case of JSON, but then we wouldn't be able to redact appropriately.